<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/image/images/icon.ico</normaloff>:/image/images/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-image:url(:/image/images/back.png)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color:white;
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
	color:white;
}</string>
         </property>
         <property name="text">
          <string>有教师上课</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
	color:white;
}</string>
         </property>
         <property name="text">
          <string>无教师上课</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leId">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:!enabled
{
	background:rgb(231, 231, 231)
}</string>
       </property>
       <property name="placeholderText">
        <string>有教师上课无需输入</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color:black;
}</string>
       </property>
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color:black;
}</string>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color:black;
}</string>
       </property>
       <property name="text">
        <string>请正确选择：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(46,46,46),stop:1 rgb(66,66,66));color:white;padding:4px;border-radius:6px;}
QPushButton:hover{background-color:rgb(80,80,80);}
QPushButton:pressed{background-color:rgb(30,30,30)}
QPushButton:!enabled
{
	background:gray;
}</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labConn">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color:rgb(255, 255, 0);
}</string>
       </property>
       <property name="text">
        <string>正在连接教师端</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color:black;
}</string>
       </property>
       <property name="text">
        <string>当前状态:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:!enabled
{
	background:rgb(231, 231, 231)
}</string>
       </property>
       <property name="placeholderText">
        <string>有教师上课无需输入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
